### PR TITLE
traefik-dynamic

### PR DESCRIPTION
**Pull request by BrianHicks**

### CODE REVIEW

File: packages/traefik-dynamic/consul-template-traefik [hunk 1]
```diff
@@ -0,0 +1,2 @@
+consul	ALL=(ALL)	NOPASSWD:	/usr/bin/systemctl restart traefik.service
+
```
Comment (stevendborrelli) [inline packages/traefik-dynamic/consul-template-traefik]: Do we need to support `systemctl daemon-reload` if the .service file changes?

Comment (BrianHicks) [inline packages/traefik-dynamic/consul-template-traefik]: That will be taken care of by yum/the package configuration, which runs 
as root and so doesn’t need special permission. Check out the upgrade 
steps in `spec.yml`

File: packages/traefik-dynamic/templates/traefik.toml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# global config
+port = "{{or (key "config/traefik/port") ":80"}}"
```
Comment (siddharthist) [inline packages/traefik-dynamic/templates/traefik.toml]: It looks like there might be a missing end backet here, Github is highlighting a lot of red.

Comment (BrianHicks) [inline packages/traefik-dynamic/templates/traefik.toml]: Github also thinks this is *just* toml, when it's toml+go template. It's a valid template which renders valid toml.

**Comment by BrianHicks** [comment]
I'm going to close this and re-evaluate if we actually need it *when* we actually need it.

**Pull request closed without merging**